pkg/cmd/chat/imports/imports.go

```diff
@@ -0,0 +1,250 @@
+
+	return cmd
+}
+
+func uploadToS3(ctx context.Context, filename, url string) error {
+	data, err := os.Open(filename)
+	if err != nil {
+		return err
+	}
+	defer data.Close()
+
+	stat, err := data.Stat()
+	if err != nil {
+		return err
+	}
+
+	req, err := http.NewRequestWithContext(ctx, "PUT", url, data)
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Content-Type", "application/json")
+	req.ContentLength = stat.Size()
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	return nil
+}
+
+func uploadCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "upload-import [filename] --mode [upsert|insert] --output-format [json|tree]",
+		Short: "Upload an import",
+		Example: heredoc.Doc(`
+			# Uploads an import and prints it as JSON
+			$ stream-cli chat upload-import data.json --mode insert
+
+			# Uploads an import and prints it as a browsable tree
+			$ stream-cli chat upload-import data.json --mode insert --output-format tree
+		`),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			filename := args[0]
+
+			results, err := validateFile(cmd.Context(), c, filename)
+			if err != nil {
+				return err
+			}
+			if results.HasErrors() {
+				return utils.PrintObject(cmd, results)
+			}
+
+			mode := stream.UpsertMode
+			if m, _ := cmd.Flags().GetString("mode"); stream.ImportMode(m) == stream.InsertMode {
+				mode = stream.InsertMode
+			}
+
+			createImportURLResp, err := c.CreateImportURL(cmd.Context(), filepath.Base(filename))
+			if err != nil {
+				return err
+			}
+
+			if err := uploadToS3(cmd.Context(), filename, createImportURLResp.UploadURL); err != nil {
+				return err
+			}
+			createImportResp, err := c.CreateImport(cmd.Context(), createImportURLResp.Path, mode)
+			if err != nil {
+				return err
+			}
+
+			return utils.PrintObject(cmd, createImportResp.ImportTask)
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringP("mode", "m", "upsert", "[optional] Import mode. Canbe upsert or insert")
+	fl.StringP("output-format", "o", "json", "[optional] Output format. Can be json or tree")
+
+	return cmd
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-import [import-id] --output-format [json|tree]",
+		Short: "Get import",
+		Example: heredoc.Doc(`
+			# Returns an import and prints it as JSON
+			$ stream-cli chat get-import dcb6e366-93ec-4e52-af6f-b0c030ad5272
+
+			# Returns an import and prints it as a browsable tree
+			$ stream-cli chat get-import dcb6e366-93ec-4e52-af6f-b0c030ad5272 --output-format tree
+		`),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			id := args[0]
+			watch, _ := cmd.Flags().GetBool("watch")
+
+			for {
```

Comment:
Should we handle infinite cycle here? Mabye not, the person can probably Command+C anyway

Reply:
yeah, it's supposed to loop until control break